src/grokcore/view/tests/functional/view/macros.py

```diff
@@ -101,38 +105,46 @@ class Painting(grok.View):
 </html>
 """)
 
+
 class Layout(grok.View):
     # Layout template is in macros_templates/layout.pt for reload test
     # purposes.
     pass
 
+
 class Dancing(grok.View):
     pass
 
+
 dancing = grok.PageTemplate("""\
 <html metal:use-macro="context/@@dancinghall/macros/something">
 </html>
 """)
 
+
 class GrillDish(grok.View):
     pass
 
+
 grilldish = grok.PageTemplate("""
 <html metal:use-macro="context/@@grilled/macros/spices">
 </html>""")
 
+
 class Burnt(grok.View):
     pass
 
+
 burnt = grok.PageTemplate("""\
 <html metal:use-macro="context/@@grilled/spices">
 </html>""")
 
-class Grilled(grok.View):
-    pass
+
+# class Grilled(grok.View):  # noqa: F811 redefinition of unused
+#     pass
```

Comment:
This is now commented. Is there a reason to keep it, if it was already defined before?

Reply:
I am going to remove it.